nixos/modules/services/networking/alice-lg.nix

```diff
@@ -0,0 +1,108 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.alice-lg;
+  settingsFormat = pkgs.formats.ini { };
+in
+{
+  options = {
+    services.alice-lg = {
+      package = mkOption {
```

Comment:
Nit (and for the next time): we have `mkPackageOptionMD` IIRC

Reply:
Changed it anyways, shouldn't really matter